src/coreclr/jit/emitxarch.cpp

```diff
@@ -15327,14 +15340,14 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             {
                 // ins   reg, mem
                 result.insThroughput = PERFSCORE_THROUGHPUT_2X;
-                // insLatency is set above (see -  Model the memory latency)
```

Comment:
Do we need to update `Model the memory latency` to not count this twice?

Reply:
This isn't counted twice, this 2/3C latency is in addition to the modeled memory access base latency (i.e., mov latency). To put it another way, `mov rax, [mem]` has lower latency than `movdqa ymm, [mem]`.